azext_edge/edge/params.py

```diff
@@ -1227,6 +1227,36 @@ def load_iotops_arguments(self, _):
             options_list=["--resource-group", "-g"],
             help="Instance resource group.",
         )
+        context.argument(
+            "include_secrets",
+            options_list=["--include-secrets"],
+            help="Indicates the command should remove the key vault secrets "
+            "associated with the certificate(s). This option will delete and "
+            "purge the secrets.",
+            arg_type=get_three_state_flag(),
+        )
+        context.argument(
+            "certificate_names",
+            options_list=["--certificate-names", "--cn"],
+            nargs="+",
+            help="Space-separated certificate names to remove. "
+            "Note: the names can be found under corresponding secretsync "
+            "resource property 'targetKey'.",
+        )
+        context.argument(
+            "force",
```

Comment:
Why does the force parameter need to be re-defined? There is no difference with this and the higher scope definition.

Reply:
oh I see, will remove